src/tiertwo/netfulfilledman.h

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2014-2020 The Dash Core developers
+// Copyright (c) 2022 The PIVX Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or https://www.opensource.org/licenses/mit-license.php.
+
+#ifndef PIVX_NETFULFILLEDMAN_H
+#define PIVX_NETFULFILLEDMAN_H
+
+#include "serialize.h"
+#include "sync.h"
+
+#include <map>
+
+class CService;
+
+static const std::string NET_REQUESTS_CACHE_FILENAME = "mnetrequests.dat";
```

Comment:
```suggestion
static const std::string NET_REQUESTS_CACHE_FILENAME = "netrequests.dat";
```
according to `strDBName` in `LoadTierTwo`